site/it/tutorials/quickstart/advanced.ipynb

```diff
@@ -0,0 +1,418 @@
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "7NAbSZiaoJ4z"
+      },
+      "source": [
+        "Carica e prepara il [dataset MNIST](http://yann.lecun.com/exdb/mnist/)."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "JqFRS6K07jJs",
+        "colab": {}
+      },
+      "source": [
+        "mnist = tf.keras.datasets.mnist\n",
+        "\n",
+        "(x_train, y_train), (x_test, y_test) = mnist.load_data()\n",
+        "x_train, x_test = x_train / 255.0, x_test / 255.0\n",
+        "\n",
+        "# Add a channels dimension\n",
+        "x_train = x_train[..., tf.newaxis]\n",
+        "x_test = x_test[..., tf.newaxis]"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "k1Evqx0S22r_"
+      },
+      "source": [
+        "Usa `tf.data` per raggruppare e mischiare il dataset:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "8Iu_quO024c2",
+        "colab": {}
+      },
+      "source": [
+        "train_ds = tf.data.Dataset.from_tensor_slices(\n",
+        "    (x_train, y_train)).shuffle(10000).batch(32)\n",
+        "\n",
+        "test_ds = tf.data.Dataset.from_tensor_slices((x_test, y_test)).batch(32)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "BPZ68wASog_I"
+      },
+      "source": [
+        "Costrusci il modello `tf.keras` usando l'[API Keras per creare sottoclassi di modelli](https://www.tensorflow.org/guide/keras#model_subclassing):"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "h3IKyzTCDNGo",
+        "colab": {}
+      },
+      "source": [
+        "class MyModel(Model):\n",
+        "  def __init__(self):\n",
+        "    super(MyModel, self).__init__()\n",
+        "    self.conv1 = Conv2D(32, 3, activation='relu')\n",
+        "    self.flatten = Flatten()\n",
+        "    self.d1 = Dense(128, activation='relu')\n",
+        "    self.d2 = Dense(10, activation='softmax')\n",
+        "\n",
+        "  def call(self, x):\n",
+        "    x = self.conv1(x)\n",
+        "    x = self.flatten(x)\n",
+        "    x = self.d1(x)\n",
+        "    return self.d2(x)\n",
+        "\n",
+        "# Create an instance of the model\n",
+        "model = MyModel()"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "uGih-c2LgbJu"
+      },
+      "source": [
+        "Scegli un metodo di ottimizzazione e una funzione di perdita per l'allenamento:"
```

Comment:
Funzione di perdita doesn't sound so good. Also addestramento IMHO sound Better and It Is more common use.

Reply:
I have found the equivalent for the loss function in wikipedia: https://it.wikipedia.org/wiki/Funzione_obiettivo
If you agree will use "Funzione obiettivo" in its place. Meanwhile I change allenamento to addestramento.